spec/support/corpus.rb

```diff
@@ -80,21 +82,11 @@ def verify_mutation_generation
           start:        method(:start),
           in_processes: parallel_processes
         }
+
         total = Parallel.map(effective_ruby_paths, options) do |path|
```

Comment:
Yeah I could rewrite that way. 

Regarding reduce: I'm just following the rubocop preferences for mutant. If @mbj prefers reduce I can update the config accordingly and make the change